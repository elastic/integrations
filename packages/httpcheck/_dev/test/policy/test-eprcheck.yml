vars:
  endpoints:
    - https://epr.elastic.co
