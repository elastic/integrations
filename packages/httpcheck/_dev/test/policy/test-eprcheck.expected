connectors:
    forward: {}
exporters:
    elasticsearch/componentid-0:
        endpoints:
            - https://elasticsearch:9200
inputs: []
output_permissions:
    default:
        _elastic_agent_checks:
            cluster:
                - monitor
        _elastic_agent_monitoring:
            indices: []
        uuid-for-permissions-on-related-indices:
            indices:
                - names:
                    - metrics-*-*
                  privileges:
                    - auto_configure
                    - create_doc
processors:
    transform/componentid-0:
        metric_statements:
            - context: datapoint
              statements:
                - set(attributes["data_stream.type"], "metrics")
                - set(attributes["data_stream.dataset"], "httpcheck.httpcheckreceiver")
                - set(attributes["data_stream.namespace"], "ep")
receivers:
    httpcheck/componentid-0:
        collection_interval: 1m
        targets:
            - endpoints:
                - https://epr.elastic.co
              method: GET
secret_references: []
service:
    pipelines:
        metrics:
            exporters:
                - elasticsearch/componentid-0
            receivers:
                - forward
        metrics/componentid-0:
            exporters:
                - forward
            processors:
                - transform/componentid-0
            receivers:
                - httpcheck/componentid-0
